Doc/library/datetime.rst

```diff
@@ -360,29 +420,34 @@ Instance methods:
 
    .. versionadded:: 3.2
 
+:class:`timedelta`: Examples
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Example usage:
+An additional example of normalization::
 
     >>> from datetime import timedelta
     >>> year = timedelta(days=365)
     >>> another_year = timedelta(weeks=40, days=84, hours=23,
-    ...                          minutes=50, seconds=600)  # adds up to 365 days
-    >>> year.total_seconds()
-    31536000.0
+    ...                          minutes=50, seconds=600)
+    >>> # Components of another_year add up to exactly 365 days
     >>> year == another_year
     True
+    >>> year.total_seconds()
+    31536000.0
+
+Examples of :class:`timedelta` arithmetic::
+
+    >>> from datetime import timedelta
+    >>> year = timedelta(days=365)
     >>> ten_years = 10 * year
-    >>> ten_years, ten_years.days // 365
-    (datetime.timedelta(days=3650), 10)
+    >>> ten_years.days // 365
```

Comment:
Since this is no longer written as a tuple, let's add lines between 442 and 443:
```
>>> ten_years
datetime.timedelta(days=3650)
```
This will make the result on line 447 easier to understand.